### PR TITLE
fixed float/double to float32, float64 conversion

### LINKED ISSUE #100: float/double instead of float32/float64
webknossos expects the datatypes to be called `float` and `double` instead of `float32` or `float64` in the `datasource-properties.json`. This needs to be converted both ways in `wkcuber.metadata`

### PR DESCRIPTION
**Pull request by MartinBuessemeyer**
This PR fixes the bug described in #100. Additionally I fixed two more bugs. An argument was missing in the tests. `read_read_metadata_for_layer` could not read a layer with the `elementClass` of `uint24` which can be produced by writing a layer with dtype `uint8` and 3 channels. I fixed it by returning `uint8` when there is no `numpy-dtype` that matches `elementClass`. 

**Steps to test:**
       run the tests

**Issues**:
      fixes #100

### CODE REVIEW

File: tests/test_metadata.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    read_datasource_properties,
+    read_metadata_for_layer,
+)
+
+
+def test_element_class_convertion():
+    test_wkw_path = os.path.join("testoutput", "test_metadata")
+    prediction_layer_name = "prediction"
+    prediction_wkw_info = WkwDatasetInfo(
+        test_wkw_path, prediction_layer_name, np.float32, 1
```
Comment (philippotto) [inline tests/test_metadata.py]: Prediction layers usually have three channels which you could reflect here.

File: tests/scripts/simple_anisotropic_tiff_cubing.sh [hunk 2]
```diff
@@ -10,6 +10,7 @@ docker run \
   --batch_size 8 \
   --layer_name color \
   --max 8 \
+  --scale "11.24,11.24,25"\
```
Comment (philippotto) [inline tests/scripts/simple_anisotropic_tiff_cubing.sh]: Do you know why the CI passed even before this change? Isn't this called in the CI?

File: wkcuber/metadata.py [hunk 3]
```diff
@@ -149,6 +161,14 @@ def read_metadata_for_layer(wkw_path, layer_name):
     return layer_info, dtype, bounding_box, origin
 
 
+def convert_dype_to_element_class(dtype):
+    convertion_map = {np.float32: "float", np.float64: "double"}
```
[the remark this answers is not in the document]
Comment (MartinBuessemeyer) [reply wkcuber/metadata.py]: That does not work any more after merging `known_nupy_types`.

File: tests/scripts/simple_anisotropic_tiff_cubing.sh (hunk 2 above)
Comment (MartinBuessemeyer) [reply tests/scripts/simple_anisotropic_tiff_cubing.sh]: Sry I don't know.

File: tests/test_metadata.py (hunk 1 above)
Comment (MartinBuessemeyer) [reply tests/test_metadata.py]: The `1` is the `Mag` and the `WKWDatasetInfo` does not contain a `num_channel`-variable.

**Pull request merged**